### DISCUSSION TITLE
wire:current is not working when manually building Livewire

### DISCUSSION THREAD

**Discussion by tpetry** [livewire/livewire#9082] [7. Bugs (archived)] [answered]
### Livewire version

3.5.17

### Laravel version

11.31.0

### Which PHP version are you using?

PHP 8.3

### What is the problem?

The `wire:current` directive does not work when [manually building Livewire and Alpine](https://livewire.laravel.com/docs/installation#manually-bundling-livewire-and-alpine). I am doing this for better asset caching and storing the build on a CDN.

### Code snippets

```blade
<nav>
    <a href="/dashboard" ... wire:current="font-bold text-zinc-800">Dashboard</a>
    <a href="/posts" ... wire:current="font-bold text-zinc-800">Posts</a>
    <a href="/users" ... wire:current="font-bold text-zinc-800">Users</a>
</nav>
```

### Wirebox URL

not possible to do this in wirebox

### How do you expect it to work?

The `wire:current` directive should style the links. Adding `@livewireScripts` to the layout does not work as it will interfere with the manual Livewire build.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by danie-ramdhani** [comment, marked as answer]
i've tried it then build the assets and it works, it impossible that doesn't work if the dist version is work, can you confirm your livewire is version [^3.5.13](https://github.com/livewire/livewire/releases/tag/v3.5.13)? it just currently doesn't work when livewire morph the page (see https://github.com/livewire/livewire/discussions/9074#discussioncomment-11544286)

**Reply by tpetry** [reply]
I don't know what the problem was. I built a completely new project to share a repo that its not working and it worked. Then I tried it again in my project and it worked today too 🤷‍♂️
